Firmware/sound.cpp

```diff
@@ -84,13 +115,43 @@ switch(eSoundMode)
           if(eSoundType==e_SOUND_TYPE_StandardAlert)
                Sound_DoSound_Alert(true);
           break;
-     case e_SOUND_MODE_MUTE:
-          break;
+     case e_SOUND_MODE_BLIND:
+          if(eSoundType==e_SOUND_TYPE_ButtonEcho)
+               Sound_DoSound_Echo();
+          if(eSoundType==e_SOUND_TYPE_StandardPrompt)
+               Sound_DoSound_Prompt();
+          if(eSoundType==e_SOUND_TYPE_StandardAlert)
+               Sound_DoSound_Alert(false);
+          if(eSoundType==e_SOUND_TYPE_EncoderMove)
+               Sound_DoSound_Encoder_Move();
+          if(eSoundType==e_SOUND_TYPE_BlindAlert)
+               Sound_DoSound_Blind_Alert();
+               break;
      default:
-          ;
+          break;
      }
 }
 
+static void Sound_DoSound_Blind_Alert(void)
+{
+     _tone(BEEPER,100);
+     delay_keep_alive(50);
```

Comment:
This is called from lcd_update() and it calls lcd_update() leading to possible stack overflow.